404.md

```diff
@@ -0,0 +1,5 @@
+[comment]: <> (This is the 404 page as used on the Github Pages site.)
+
+## 404 Page Not Found - The Rose Documentation Has Moved
+
+Please update your bookmarks to: http://metomi.github.io/rose
```

Comment:
We could write this file in `rose make-docs` if we don't want it kicking about in the repo.

Reply:
I think it is fine as it is.